src/Draco.Compiler.Benchmarks/SyntaxBenchmarks.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;
+using Draco.Compiler.Api.Syntax;
+using Draco.Compiler.Internal.Syntax;
+using SyntaxToken = Draco.Compiler.Internal.Syntax.SyntaxToken;
+
+namespace Draco.Compiler.Benchmarks;
+
+public class SyntaxBenchmarks : FolderBenchmarkBase
```

Comment:
Note for later: 
IIRC BDN by default "compare" the methods of a benchmark, we may need to config something to tell it it's apple and oranges and it's useless to compare those.